src/aspect.ts

```diff
@@ -281,7 +281,7 @@ function getDispatcherObject(target: Targetable, methodName: string): Dispatcher
 function getJoinPointDispatcher<F extends GenericFunction<T>, T>(joinPoint: F): F {
 
 	function dispatcher(this: Function, ...args: any[]): T {
-		const { before, after, joinPoint } = dispatchAdviceMap.get(dispatcher);
+		const { before, after, joinPoint } = dispatchAdviceMap.get(dispatcher)!;
```

Comment:
I was debating this, since from the code below, you could never get a situation where the map for the `dispatcher` is `undefined`, because the only way the way the function could be called is if its map had been set.